### PR TITLE
Adding facebook-share tracking.

### LINKED ISSUE #27: Add shared event on facebook-share
Hey guys, thanks for the nice plugin.

Is there any plan to add the shared event on facebook-share?

Can we add the event on this line?

https://github.com/plyfe/ember-social/blob/38cea5b8bd1a78a217f2074dac8a4aed564a148a/addon/components/facebook-share.js#L60

I know facebook send the same response even if the dialog was closed, but for now I need this in a situation I have and maybe others can have the same case. Also if the user is logged-in on your app facebook then confirm the post with a post_id on the response.

If it's ok and a PR helps, lemme know!!

### PR DESCRIPTION
**Pull request by denneralex**
Resolve #27.

### CODE REVIEW

File: tests/dummy/app/templates/tracking.hbs [hunk 1]
```diff
@@ -85,6 +85,6 @@
...
 
-  <p>* The <code>\{{facebook-share}}</code> component is is subject to the Facebook SDK's restrictions as to what can be tracked. Specifically, click tracking is only
-  available when using the component with <code>tagName="a"</code> (i.e. not the standard Facebook UI), and share tracking is not available. (In theory, this addon could support share tracking in the case that the user grants or has previously granted your Facebook app <code>publish_action</code> permission.)</p>
+  <p>* The <code>\{{facebook-share}}</code> component is is subject to the Facebook SDK's restrictions as to what can be tracked. Specifically, click and share tracking is only
+  available when using the component with <code>tagName="a"</code> (i.e. not the standard Facebook UI), and share tracking is only available in the case that the user grants or has previously granted your Facebook app <code>publish_action</code> permission. Facebook calls the callback on share even if the user click on the close button on the dialog but if you have the permission from the user and the post was sucessful facebook sends the post_id on the payload.</p>
```
Comment (lukemelia) [inline tests/dummy/app/templates/tracking.hbs]: s/user click on/user clicks on/

**Comment by denneralex** [comment]
Hey guys! Any plan to merge this?

**Comment by lukemelia** [comment]
Thanks @denneralex! Sorry for the delay.

**Comment by denneralex** [comment]
Tks guys!!

**Pull request merged**